### ISSUE TITLE
Add examples with manually-launched components

### ISSUE THREAD

**Issue by Vayel** [crossbario/crossbarwww#24]
On the Crossbar.io Web Site, there are only examples with components launched from the `config.json` file. Adding some examples with manually-launched components would both make easier for us to understand that Crossbar.io is just a router, that the components are independent of it and give us a way to launch our components manually.

Thanks and sorry for my English.

**Comment by goeddea** [comment]
First of all  - thanks for contributing (and I really wonder why you worry about your English - those are two well-constructed sentences).

To help us improve the documentation, could you be a bit more precise: Where exactly do you think we should point out that components can, of course, be launched independently of Crossbar.io? To me the fact that basically every example has a Web frontend, which is not hosted by Crossbar.io, but necessarily runs in the browser, already shows that. But then that's me, and my perspective is different from somebody coming to the documentation for the first time.

**Comment by Vayel** [comment]
Sorry, I was not explicit enough: I mainly refered to Python components. In the documentation, they are launched by Crossbar.io, due to those lines in the `config.json` file: 

```javascript
{
    "type": "container",
    "options": {
        "pythonpath": [".."]
    },
    "components": [
        {
            "type": "class",
            "classname": "hello.hello.AppSession",
            "realm": "realm1",
            "transport": {
                "type": "websocket",
                "endpoint": {
                    "type": "tcp",
                    "host": "127.0.0.1",
                    "port": 8080
                },
                "url": "ws://127.0.0.1:8080/ws"
            }
        }
    ]
}
```

Perhaps an example using the `ApplicationRunner` class would help to understand that it's possible to launch Python components without adding them in the `config.json` file.

**Comment by goeddea** [comment]
So basically make this explicit in the context, and (for now) link to http://autobahn.ws/python/wamp/programming.html which gives information on this?

**Comment by Vayel** [comment]
Linking to Autobahn website will indeed do the job in the short term. If #26 is developped, a step-by-step example explaining how to write basic components, launch them and then connect them with Crossbar.io will be more explicit.

**Comment by goeddea** [comment]
Ok, so closing this. Think what you wanted (short term) is fixed, and #26 covers the broader issue.
